### PR TITLE
Added Loss Function & Rewrote Unit Testing

### PR DESCRIPTION
**Pull request by shibzhou**
Update:
1. Added RMAELoss function in Pytorch and its unit testing. 
2. Fixed MASELoss in Pytorch. 
3. Rewrote unit testing files.

### CODE REVIEW

**Comment by kdgutier** [comment]
Thanks

**Review by kdgutier — APPROVED** [review]
